File: backend/manga-ln-app/src/main/java/com/example/simeon/manga_ln_app/dto/ContentBetaDTO.java
package com.example.simeon.manga_ln_app.dto;

import com.example.simeon.manga_ln_app.models.ContentType;
import lombok.Data;

@Data
public class ContentBetaDTO {
    private int id;
    private ContentType contentType;
    private String name;
    private String authorUsername;
} 

File: mobile-app/app/src/main/java/com/example/manga_ln_app/data/dto/ContentBetaDto.kt
package com.example.manga_ln_app.data.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ContentBetaDto(
    @SerialName("id") val id: Int,
    @SerialName("contentType") val contentType: String,
    @SerialName("name") val name: String,
    @SerialName("authorUsername") val author: String
)
